app/screens/Course/Course18.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { exchangeSlice } from '../../redux/slices';
+import { HomeScreen } from '..';
+import { styles } from './course.styles';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { Course } from './Course';
+
+export const Course18 = () => {
+  const data18 = '01.01.2018';
```

Comment:
The same - move it away from component - in another file called `Charts18.data.ts`